CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ This document describes changes between each past release.
   ``instance_uri_registry``, suitable for use when you don't
   necessarily have a ``request`` object around. The existing functions
   will remain in place.
+- openid plugin will carry ``prompt=none`` query string parameter if appended
+  to authorize endpoint.
```

Comment:
If you still have some energy we could mention how to use this in the openid docs section ? :) :)